### ISSUE TITLE
Spectest failures t/spec/S26-documentation/07-tables.t

### ISSUE THREAD

**Issue by zoffixznet** [rakudo/rakudo#1275]
Seems to be something with POD and tables. cc @tbrowder 

```
$ ./perl6 -v
This is Rakudo version 2017.11-37-gca7d048 built on MoarVM version 2017.11
implementing Perl 6.c.
```

```
$ t/fudgeandrun t/spec/S26-documentation/07-tables.t
t/spec/S26-documentation/07-tables.t .. 
===WARNING: One or more tables evidence bad practice.
==          Set environment variable 'RAKUDO_POD6_TABLE_DEBUG' for more details.
1..42
ok 1 - The object is-a 'Pod::Block::Table'
ok 2 - 
ok 3 - 
ok 4 - 
ok 5 - 
ok 6 - 
ok 7 - 
ok 8 - 
ok 9 - 
ok 10 - 
ok 11 - 
ok 12 - 
ok 13 - 
ok 14 - 
ok 15 - 
ok 16 - 
ok 17 - 
ok 18 - 
ok 19 - 
ok 20 - 
ok 21 - 
ok 22 - 
ok 23 - 
ok 24 - 
ok 25 - 
ok 26 - 
ok 27 - 
ok 28 - 
ok 29 - 
ok 30 - 
ok 31 - ensure trailing whitespace counts as a cell (WARNING: this test will fail if you modified this file and your editor auto-stripped trailing whitespace)
ok 32 - 
ok 33 - 
ok 34 - 
ok 35 - ensure trailing whitespace counts as a cell (WARNING: this test will fail if you modified this file and your editor auto-stripped trailing whitespace)
ok 36 - 
ok 37 - 
ok 38 - 
ok 39 - 
ok 40 - 
ok 41 - 
not ok 42 - 
# Failed test at t/spec/S26-documentation/07-tables.t line 164
# expected: 'x,y'
#      got: 'x,y,'
# Looks like you failed 1 test of 42
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/42 subtests 

Test Summary Report
-------------------
t/spec/S26-documentation/07-tables.t (Wstat: 256 Tests: 42 Failed: 1)
  Failed test:  42
  Non-zero exit status: 1
Files=1, Tests=42,  0 wallclock secs ( 0.01 usr  0.01 sys +  0.53 cusr  0.11 csys =  0.66 CPU)
Result: FAIL
```

**Comment by tbrowder** [comment]
The recently-merged NQP pod code (rakudo PR #1240) requires roast PR #353 for new and corrected tests. @AlexDaniel should approve that merge soon (hopefully).

**Comment by AlexDaniel** [comment]
Should be ok now.
